### ISSUE TITLE
Does not support camel case css property in jsx

### ISSUE THREAD

**Issue by bin-san** [solidjs/solid#2034]
### Describe the bug

<div style = {{backgroundColor = "black"}}></div> has no effect
but <div style = {{"background-color" = "black"}}></div> works.
but former supports suggestions in vscode but the later do not....

### Your Example Website or App

na

### Steps to Reproduce the Bug or Issue

na

### Expected behavior

na

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by lxsmnsyc** [comment]
lack of support for camel-case is by design.

**Comment by ryansolid** [comment]
Yeah this is intentional. Originally setup here: https://github.com/solidjs/solid/issues/143

While this doesn't preclude potentially supporting camelCase there are a lot of benefits of not making it the primary method. And having 2 ways to do things is not great.
